packages/azure-services/src/azure-cosmos/cosmos-client-wrapper.ts

```diff
@@ -17,11 +18,15 @@ export class CosmosClientWrapper {
     public async upsertItems<T>(items: T[], dbName: string, collectionName: string, partitionKey?: string): Promise<void> {
         const container = await this.getContainer(dbName, collectionName);
 
-        await Promise.all(
-            items.map(async item => {
-                await container.items.upsert(item, this.getOptions(item, partitionKey));
-            }),
-        );
+        const chunks = System.chunkArray(items, 10);
```

Comment:
we hit tcp error when sending 1000 requests in a call.

Exception: Error: connect ETIMEDOUT <ip address>
Stack: Error: connect ETIMEDOUT  <ip address>
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14) 

referred https://github.com/Azure/azure-storage-node/issues/343